core/build.gradle.kts

```diff
@@ -35,10 +35,6 @@ dependencies {
     implementation("net.java.dev.jna:jna:5.5.0")
     api("de.swirtz:ktsRunner:0.0.7") { exclude(group = "ch.qos.logback", module = "logback-classic") }
 
-    // AutoDsl Annotation
-    api("com.juanchosaravia.autodsl:annotation:0.0.9")
```

Comment:
Maybe     `kotlin("kapt")` in the begining of the file can also be removed

Reply:
Yes we can delete it safely

Reply:
removed in commit c8dc78da6b50da62a500c674bae2a1719fa24d23